index.js

```diff
@@ -0,0 +1,53 @@
+// Set the date we're counting down to
+var countDownDate = new Date("Jan 29, 2023 12:48:00").getTime();
```

Comment:
Suggestions: You can move this whole logic in separated file for good structure.